pyv/inference.py

```diff
@@ -11,7 +11,7 @@
 
 V_TYPE_MAP: Dict[type, str] = {
     int: "int",
-    float: "f32",
+    float: "f64",
```

Comment:
@div72 , is this change going to cause you any problems?  `f64` is closer to what `float` is in Python, but I dont want to disrupt your use of py2many if it can be avoided, and iirc this change isnt needed for the `float(1)` test to pass.

Reply:
Nope, feel free to. I decided to go with f32 to lessen the size of structs IIRC.